src/axom/inlet/inlet_utils.hpp

```diff
@@ -10,16 +12,40 @@ namespace inlet
 
 
 /*!
-  *****************************************************************************
-  * \brief This function is used to mark if anything went wrong during the 
-  * defining phase of inlet so verify() will properly fail.
-  *
-  * \param [in] root Pointer to the Sidre Root Group where the warning flag 
-  * will be set.
-  *****************************************************************************
-  */
+*****************************************************************************
+* \brief This function is used to mark if anything went wrong during the 
+* defining phase of inlet so verify() will properly fail.
+*
+* \param [in] root Pointer to the Sidre Root Group where the warning flag 
+* will be set.
+*****************************************************************************
+*/
 void setWarningFlag(axom::sidre::Group* root);
 
+/*!
+*****************************************************************************
+* \brief This function appends the prefix name to the ending name.
+*
+* \param [in] The prefix string name.
+* \param [in] The ending string name.
+*
+* \return The appended string.
+*****************************************************************************
+*/
+std::string appendPrefix(const std::string& prefix, const std::string& name);

```

Comment:
Should this function take an optional third parameter for the separator? With a default of "/" ?

Reply:
There needs to be a follow up PR that adds support for changing the default '/'